docs/releases/v1.0.0.rst

```diff
@@ -13,6 +13,9 @@
       can now be an unsorted list. The list can also contain duplicate
       values.
 
+    -[fix] Fix a crash when saving frames larger than 4Gb to jay or csv
+      on Windows. [#]
```

Comment:
If there is no corresponding issue, then `[#]` is superfluous.

Reply:
Do you use issue or PR numbers here? I was going to add a current PR number here.

Reply:
I usually use the issue number, just because PR# doesn't exist at the moment this is written. But I think it could work with the PR# too, if added later. GitHub will handle the redirect just fine.

Reply:
Yeah, that's why I left it empty and then wanted to update with the PR number... but I already created the issue, so will update it with the issue numbers.